crates/db_schema/src/impls/activity.rs

```diff
@@ -81,12 +81,9 @@ mod tests {
       .unwrap()
       .into();
 
-    // inserting activity for first time
-    let res = ReceivedActivity::create(pool, &ap_id).await;
-    assert!(res.is_ok());
-
-    let res = ReceivedActivity::create(pool, &ap_id).await;
-    assert!(res.is_err());
+    // inserting activity should only work once
+    ReceivedActivity::create(pool, &ap_id).await.unwrap();
```

Comment:
Why remove the asserts `is_ok`  and `is_err`

Reply:
They failed in CI and I needed to make this change to see the error